### PR TITLE
Stable 7.0 latest fixes

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-runtime/src/pages/account.vue [hunk 1]
```diff
@@ -77,6 +77,7 @@
         <dd data-testid="language">
           <oc-select
             v-if="languageOptions"
+            :placeholder="$gettext('Please choose...')"
```
Comment (kulmann) [inline packages/web-runtime/src/pages/account.vue:80]: We need to include translations in the PR to `stable-7.0`. Let's do the transifex-dance quickly...

**Review by kulmann — CHANGES_REQUESTED** [review]

File: packages/web-runtime/src/pages/account.vue (hunk 1 above)
Comment (kulmann) [reply packages/web-runtime/src/pages/account.vue:80]: I started a translation sync just now... next step: translate in transifex, then again translation sync, then cherry-pick the commit into this PR

Comment (JanAckermann) [reply packages/web-runtime/src/pages/account.vue:80]: where does it land ? core ?

Comment (kulmann) [reply packages/web-runtime/src/pages/account.vue:80]: all done.

**Review by kulmann — APPROVED** [review]

**Review by JammingBen — APPROVED** [review]

**Pull request merged**